modules/installation-osp-installation-parameters.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// Or not, for now.
+
+[id="installation-osp-installation-parameters_{context}"]
+= OpenStack-specific parameters
+
+In the OpenShift Installer `install-config.yaml`, you can set the following OpenStack-specific options:
+
+.OpenStack installation customization parameters
+[cols=".^2m,.^3a,^3a,options="header"]
+|====
+|Parameter|Description|Values
+
+|machines.platform.openstack.region
+|The region where the OpenStack cluster is created.
+|String, for example `region1`.
+
+|machines.platform.openstack.cloud
+|The name of the OpenStack cloud to use from `clouds.yaml`.
+|String, for example `MyCloud`.
+
+|machines.platform.openstack.externalNetwork
+|The OpenStack external network name to be used for installation.
+|String, for example `external`.
+
+|machines.platform.openstack.computeFlavor
+|The OpenStack flavor to use for control plane and compute machines.
+|String, for example `m1.xlarge`.
+
+|machines.platform.openstack.lbFloatingIP
+|An existing floating IP address to associate with the load balancer API.
+|An IP address, for example `128.0.0.1`.
+
+|machines.platform.openstack.trunkSupport
+|Whether OpenStack ports can be trunked. 
+|`true` or `false`
+
+|machines.platform.openstack.octaviaSupport
+|Whether OpenStack supports Octavia.
+|`true` or `false`
+
+|machines.platform.openstack.defaultMachinePlatform
+| _Optional_. The default configuration used when installing on OpenStack for machine pools that do not define their own platform configuration.
+| -
+
+|====
+
+For additional definitions, see https://godoc.org/github.com/openshift/installer/pkg/types/openstack#Platform[the Go documentation].
+
+While installing {product-title}, you refer to your modified `install-config.yaml` to customize the installation.
+
+== Example
```

Comment:
This should be in its own file and presented after the step to generate the install-config.yaml file.